experimental/auth-logic/provenance_build_wrapper.go

```diff
@@ -25,40 +25,44 @@ import (
 
 type provenanceBuildWrapper struct{ provenanceFilePath string }
 
-func (pbw provenanceBuildWrapper) EmitStatement() UnattributedStatement {
-	handleErr := func(err error) {
-		if err != nil {
-			panic(err)
-		}
-	}
-
+func (pbw provenanceBuildWrapper) EmitStatement() (UnattributedStatement, error) {
 	// Unmarshal a provenance struct from the JSON file.
-	provenance, provenanceParseErr := slsa.ParseProvenanceFile(pbw.provenanceFilePath)
-	handleErr(provenanceParseErr)
+	provenance, err := slsa.ParseProvenanceFile(pbw.provenanceFilePath)
+	if err != nil {
+		return UnattributedStatement{}, err
```

Comment:
Is is sufficient to return the error, or would it be useful to add additional info here as explained in https://g3doc.corp.google.com/go/g3doc/style/best-practices.md?cl=head#error-extra-info. Also, everywhere else in this PR.

Reply:
ok fixed.